src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,70 @@
+package duke.tasks;
+
+public class Task {
+    private String description;
+    private boolean status;
```

Comment:
maybe "status" could be renamed to sound more booleanish like isDone or isComplete!